uefi-sct/SctPkg/TestCase/UEFI/EFI/BootServices/ProtocolHandlerServices/BlackBoxTest/ProtocolHandlerBBTestStress.c

```diff
@@ -1792,7 +1794,7 @@ BBTestCombinationTest2CheckPoint4 (
 
       TempInfo = OpenInfo;
       Found = FALSE;
-      for (Index = 0; Index < 4; Index++) {
+      for (Index = 0; Index < TPL_ARRAY_SIZE; Index++) {

```

Comment:
Actually, after thinking this over, we should still use value 4 as a boundary because the size of TempInfo/OpenInfo buffer is 4.

Reply:
> Actually, after thinking this over, we should still use value 4 as a boundary because the size of TempInfo/OpenInfo buffer is 4, and we want to check all 4 entries in the buffer in this for loop.

Updated